syntax = "proto3";

import "DTO.proto";

package SSDTO;

message Login_DTO{
    bytes SSID = 1;
    bytes password = 2;
    bytes email = 3;
    bytes ip = 4;
    SSDTO.Business_Type type = 5;
}